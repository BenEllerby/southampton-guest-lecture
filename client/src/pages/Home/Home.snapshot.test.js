import React from 'react';
import Home from './Home.component';
import createComponentWithIntl from '../../services/i18n/create-component-with-intl';

describe('The Home component', () => {
  it('should renders correctly', () => {
    const props = {
      intl: {
        formatMessage: jest.fn(),
      },
      fetchUser: jest.fn(),
      updateUserId: jest.fn(),
    };
    const tree = createComponentWithIntl(<Home {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
