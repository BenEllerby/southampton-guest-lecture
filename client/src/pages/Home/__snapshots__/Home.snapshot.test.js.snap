// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home component should renders correctly 1`] = `
<div>
  <p
    className="intro"
  >
    <span>
      Pour commencer, modifie
    </span>
    <code
      className="intro-code"
    >
      src/App.js
    </code>
    <span>
      et sauvegarde pour recharger.
    </span>
  </p>
  <button
    onClick={[Function]}
  >
    <span>
      Click sur moi
    </span>
  </button>
  <a
    onClick={[Function]}
    style={Object {}}
  >
    <span>
      Ou utilise un lien
    </span>
  </a>
  <p>
    <span>
      My amazing app
    </span>
  </p>
</div>
`;
